scripts_src/headers/define.h

```diff
@@ -341,7 +341,13 @@
 #define HEAD_TRAY           (201)
 #define HEAD_ARTH           (202)
 #define HEAD_BRIGE          (203)
-
```

Comment:
Cindy, rodney, and guzz are not here, are they part of your companion mod?

Reply:
They are yes, I can add them to the CharacterFileReference if you wish but I imagine they are not relevant to this project.

I apologize for not mentioning them, please don't delete them though as it would cause issues to the Companion Expansion mod lol